packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -34,8 +34,10 @@
   "governanceTokenOwner": "0x5C4e7Ba1E219E47948e6e3F55019A647bA501005",
   "l2GenesisBlockGasLimit": "0x1c9c380",
   "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
-  "gasPriceOracleOverhead": 188,
-  "gasPriceOracleScalar": 684000,
+  "gasPriceOracleOverhead": 0,
+  "gasPriceOracleScalar": 0,
+  "gasPriceOracleBaseFeeScalar": 1368,
+  "gasPriceOracleBlobBaseFeeScalar": 810949,
```

Comment:
Matches mainnet, but the pre-ecotone values are set to 0, unlike in the hardhat deploy config. Is that intentional?

Reply:
When the pre values are 0, then it uses the new values